.circleci/config.yml

```diff
@@ -1044,6 +1075,8 @@ workflows:
   # Workflow for feature, bugfix, main branches
   feature-main-branch:
     jobs:
+      - licenses-check:
```

Comment:
I think we must move it after approve

Reply:
It's run weekly very rare, let it be automatic to don't forget about this